graylog2-web-interface/src/views/components/aggregationwizard/elementConfiguration/SortConfiguration.tsx

```diff
@@ -33,7 +39,9 @@ const SortConfiguration = () => {
                     <div>
                       {sort.map((s, index) => (
                         // eslint-disable-next-line react/no-array-index-key
-                        <ElementConfigurationSection key={`sort-${index}`}>
+                        <ElementConfigurationSection key={`sort-${index}`}
+                                                     onRemove={() => removeSort(index)}
```

Comment:
We can simplify this a bit by using formik's array helpers and just do this:
```suggestion
                                                     onRemove={() => remove(index)}
```

where `remove` is coming from the arguments of the render prop of `FieldArray`.

Reply:
Good point, I've implemented this change for every aggregation element. It was possible for all elements, but `GroupBy`. We mainly need the `onRemove` attribute for the `GroupBy` element, to be able to remove all form values for the `groupBy` key, when we remove the last grouping. Otherwise we would display the group by section.

We could also use the array helper for groupings, remove the `onRemove` aggregation element attribute for `GroupBy` as well and implement an aggregation element attribute which indicates if we want to display the related section. But I personally prefer to cleanup the form values for a specific element type, when we remove all related elements.